src/isIsogram.test.js

```diff
@@ -6,4 +6,31 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`True with 'isog'`, () => {
+    expect(isIsogram('isog'))
+      .toBeTruthy();
+  });
+
+  it(`False with 'isogg'`, () => {
+    expect(isIsogram('isogg'))
+      .toBeFalsy();
+  });
+
+  it(`takes only letters'`, () => {
+    expect(() => {
+      isIsogram(777);
+    })
+      .toThrow();
+  });
+
+  it(`Empty string equal "isogram"`, () => {
```

Comment:
Consider providing more descriptive test case names. For example, instead of `Empty string equal "isogram"`, you could use `returns true for an empty string`.